fix(cketh): Proposal to replace Ankr in the ckETH minter (#1430)
dfinity/ic: rs/ethereum/cketh/mainnet/minter_upgrade_2024_09_11.md

# Proposal to upgrade the ckETH minter canister

Git hash: `dfb1c634d08ec2248feb4d5792554bbe43e068c7`

> New compressed Wasm hash: `1a19eb2e54d5a7369ed13c6bdfe795429c8a476aac9e8d8e0cda3bee3d861271`

Reviewer: `1a19eb2e54d5a7369ed13c6bdfe795429c8a476aac9e8d8e0cda3bee3d861271` confirmed

Reviewer 2: wasm hash confirmed.

Target canister: `sv3dd-oaaaa-aaaar-qacoa-cai`

Previous ckETH minter proposal: https://dashboard.internetcomputer.org/proposal/132132

---

## Motivation
The Ethereum JSON-RPC provider Ankr (`rpc.ankr.com`) recently dropped its IPv6 connectivity and will need according to its support team a month to fix it.
This resulted in the ckETH minter being stuck and unable to process deposits nor withdrawals.

> As an emergency fix, this proposal replaces Ankr by another provider: `eth-pokt.nodies.app` from [Pocket Network](https://www.pokt.network/).

Reviewer: ```suggestion
As a temporary fix, this proposal replaces Ankr by another provider: `eth-pokt.nodies.app` from [Pocket Network](https://www.pokt.network/).
```

The long term solution is to use a more robust strategy (e.g., agreement among 3 providers, when 4 were queried) using the EVM-RPC canister.

## Upgrade args

```
git fetch
git checkout dfb1c634d08ec2248feb4d5792554bbe43e068c7
cd rs/ethereum/cketh/minter

> didc encode '()'

Author: The usual
```
4449444c0000
```

Reviewer: 4449444c0000
